### ISSUE TITLE
MOUSE_MODE_CAPTURED not working as expected if mouse is outside of game window when game starts

### ISSUE THREAD

**Issue by Krumelur** [godotengine/godot#44050]
**Godot version:**
3.2.3.stable.official

**OS/device including version:**
macOS Big Sur

**Issue description:**
I set the mouse mode:

```
func _ready():
	Input.set_mouse_mode(Input.MOUSE_MODE_CAPTURED)
```

and observe mouse movement and clicks only when the mouse is captured:

```
if Input.get_mouse_mode() == Input.MOUSE_MODE_CAPTURED:
...
```

If the mouse cursor is inside the rectangle of the screen where the game window is going to be displayed, all is working fine. If, however, the mouse cursor is outside of the rectangle, the mouse will be completely captured: the cursor disappears as expected, but clicks still get picked up by the window under the cursor (the game editor, typically).    

Expected: since the mode is captured, mouse button clicks should be handled by the game window (just like movement events do)

**Steps to reproduce:**

![Screenshot 2020-12-02 at 22 45 37 (2)](https://user-images.githubusercontent.com/574373/100936036-366fee80-34f1-11eb-8c9b-79b4e9cc6229.png)

**Minimal reproduction project:**
---

**Comment by aaronfranke** [comment]
Interesting, this does not seem to match the behavior described by @name-here, uh, [here](https://github.com/godotengine/godot/pull/43450#issuecomment-727345123) (even though both are incorrect).

> When captured mode is activated, the cursor is warped to the center of the screen, but the mouse position (as printed in your example project) does not update until I attempt to move the mouse. This seems like a minor bug.

> Current master seems to have all of the oddities I mentioned above

Can you test this on the master branch? This might be fixed on master. Anyway, adding "needs testing".

**Comment by bruvzg** [comment]
#42321 duplicate, should be fixed in master and 3.2.4 betas.

**Comment by name-here** [comment]
> Interesting, this does not seem to match the behavior described by @name-here, uh, [here](https://github.com/godotengine/godot/pull/43450#issuecomment-727345123) (even though both are incorrect).

I'm on macOS Catalina, so it might be a difference between OS versions?  But I also only tested on that PR and the master branch, not on 3.2.3 stable.
